docs/rest_api.md

```diff
@@ -1310,3 +1330,31 @@ Remove a user from a project.
 Authorization requirements:
 
 * Administrative access.
+
+#### get_status
```

Comment:
It might be nice to (1) provide more information about the action (type, nic, network etc), and (2) make this similar to the (other) `show_*` calls. So this would be something like `show_networking_action`, the URL should probably be e.g. `/networking_action/<label>`. The only difference here is that the label is chosen by HIL rather than the caller.